roles/openshift_sdn/files/sdn.yaml

```diff
@@ -86,6 +86,8 @@ spec:
           # Take over network functions on the node
           rm -Rf /etc/cni/net.d/80-openshift-network.conf
           cp -Rf /opt/cni/bin/* /host/opt/cni/bin/
+          # remove versions of the sdn binaries for old pods
+          rm -f /opt/cni/bin/openshift-sdn-*
           cp -f /opt/cni/bin/openshift-sdn /host/opt/cni/bin/openshift-sdn-${OPENSHIFT_SDN_POD}
```

Comment:
@JacobTanenbaum I totally lost context of why we copy the sdn binary to ${OPENSHIFT_SDN_POD}

But also, the rm is from /opt but the piled up binaries are going to be in /host/opt right?

The script already copied everything in /opt/cni/bin* to the host, but /opt is the container's filesystem and will be fresh every time. IIRC we want to prevent binaries from piling up in /host/opt though.

Reply:
we are tagging the openshift-sdn binary with the pods name because we saw issues with evicting pods deleting the openshift-sdn binary after the new sdn pod copied and expected the binary to be present